src/client.ts

```diff
@@ -1,5 +1,8 @@
 import { request } from 'http';
-import { createWriteStream } from 'fs';
+import { createWriteStream, readFileSync } from 'fs';
+import data from './data.json';
+
+const userPassword = process.env.SERVER_PASSWORD || 'qvfWwe3';
```

Comment:
no hay necesidad de poner el fallback

```suggestion
const userPassword = process.env.SERVER_PASSWORD;
```

Reply:
Ya lo corregí y los cambios han sido subidos.